grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -757,8 +759,13 @@ class InventoryItemController {
             if (!itemInstance.hasErrors() && itemInstance.save(flush: true)) {
                 flash.message = "${warehouse.message(code: 'default.updated.message', args: [warehouse.message(code: 'inventoryItem.label', default: 'Inventory item'), itemInstance.id])}"
             } else {
-                flash.message = "${warehouse.message(code: 'default.not.updated.message', args: [warehouse.message(code: 'inventoryItem.label', default: 'Inventory item'), itemInstance.id])}"
+                flash.error = "${warehouse.message(code: 'default.not.updated.message', args: [warehouse.message(code: 'inventoryItem.label', default: 'Inventory item'), itemInstance.id])}"
                 log.info "There were errors trying to save inventory item " + itemInstance?.errors
+                if (params.expirationDate < date) {
+                    flash.error = "This date is invalid. Please enter a date after 2000."
```

Comment:
If you moved min expiration date to config, then value in this error should also be configurable / taken from the same property